app.js

```diff
@@ -26,6 +26,61 @@ You'll respond to those questions in a professional way.
 When you include markdown text, convert them to Slack compatible ones.
 When a prompt has Slack's special syntax like <@USER_ID> or <#CHANNEL_ID>, you must keep them as-is in your response.`;
 
+function createFeedbackBlock() {
+  const elements = [
+    {
+      type: 'feedback_buttons',
+      action_id: 'feedback',
+      positive_button: {
+        text: { type: 'plain_text', text: 'Good Response' },
+        accessibility_label: 'Submit positive feedback on this response',
+        value: 'good-feedback',
+      },
+      negative_button: {
+        text: { type: 'plain_text', text: 'Bad Response' },
+        accessibility_label: 'Submit negative feedback on this response',
+        value: 'bad-feedback',
+      },
+    },
+  ];
+  return {
+    type: 'context_actions',
+    elements: elements,
+  };
+}
+app.action('feedback', async ({ ack, body, client, logger }) => {
```

Comment:
suggestion: I'd add a newline between before the `app.action(...` function. It would also be nice to have a comment above it since we document the other methods.
```suggestion
}

app.action('feedback', async ({ ack, body, client, logger }) => {
```

Reply:
Commit 1981996 adds the newline